.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Each line is a file pattern followed by one or more owners (@username or
+# email@address).
+#
+# Order is important. The last matching pattern has the most precedence.
+# So if a pull request only touches javascript files, only these owners
+# will be requested to review.
+
```

Comment:
Is it possible to use GitHub teams instead of individual people (where possible)?

Reply:
Apparently I can do this (the documentation I had was somewhat outdated and it wasn't possible at the time).